usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -369,7 +369,7 @@ def set_if_found(dl_name, path, model, file):
             'funding_sub_agency_name': 'latest_transaction__assistance_data__funding_sub_tier_agency_na',
             'last_modified_date': 'latest_transaction__assistance_data__modified_at',
             'non_federal_funding_amount': 'latest_transaction__assistance_data__non_federal_funding_amount',
-            'original_subsidy_cost': 'latest_transaction__assistance_data__original_loan_subsidy_cost',
```

Comment:
Does updating this file update all mappings for both bulk & award search download?

Reply:
Yes, this file is used for both award search and bulk download.  This field specifically is for award downloads assistance files.